src/server/req_signal.c

```diff
@@ -112,19 +113,20 @@ req_signaljob(struct batch_request *preq)
 	if (parent == (job *)0)
 		return;		/* note, req_reject already called */
 
-	if ((strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_RESUME) == 0) ||
-		(strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_SUSPEND) == 0)) {
+	if (strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_RESUME) == 0 || strcmp(preq->rq_ind.rq_signal.rq_signame, SIG_ADMIN_RESUME) == 0)
+		susrsm = -1;
```

Comment:
Why not use two different variables (suspend and resume) like you have done in req_signaljob2 ? Or change req_signaljob2 to use a single variable called susrsm - only in the interest of similarity